prac_07/project_management.py

```diff
@@ -11,6 +11,7 @@
 class_projects = []
 project_dictionary = {}
 
+
```

Comment:
Overall, it is acceptable, that you did the sorting and separating the completed and incomplete project, so make sure the method is in your own definition class.